code/web/src/modules/user/Profile.js

```diff
@@ -18,82 +18,120 @@ import { logout } from './api/actions'
 import { APP_URL } from '../../setup/config/env'
 
 // Component
-const Profile = (props) => (
-  <div>
-    {/* SEO */}
-    <Helmet>
-      <title>My Profile - Crate</title>
-    </Helmet>
-
-    {/* Top title bar */}
-    <Grid style={{ backgroundColor: grey }} justifyRight={true}>
-      <GridCell style={{ padding: '2em', textAlign: 'center', maxWidth: '20vw', marginRight: '24%' }}>
-        <H3 font="secondary">My profile</H3>
-      </GridCell>
-      
-      <GridCell style={{ padding: '2em', textAlign: 'center', maxWidth: '9vw' }}>
-        <Button theme="secondary" onClick={props.logout} style={{ marginLeft: '1em' }}>Logout</Button>
-      </GridCell>
-    </Grid>
-
-  
-    <Grid>
-      <GridCell style={{ padding: '2em', textAlign: 'center' }}>
-        <img
-          src={`${APP_URL}//images/stock/men/4.jpg`}
-          style={{ 
-            marginBottom: '2em', 
-            height: '24em', 
-            boxShadow: level1, 
-            borderRadius: '10px'
+const Profile = ({ user, logout }) => {
+  const { details } = user
+
+  return (
+    <div>
+      {/* SEO */}
+      <Helmet>
+        <title>My Profile - Crate</title>
+      </Helmet>
+
+      {/* Top title bar */}
+      <Grid style={{ backgroundColor: grey }} justifyRight={true}>
```

Comment:
What is the point of the double wrapped object notation here around the styling element?